src/cmdc_tools/datasets/official/CO/data.py

```diff
@@ -0,0 +1,124 @@
+import pandas as pd
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import ArcGIS
+
+
+class Colorado(DatasetBaseNoDate, ArcGIS):
+    ARCGIS_ID = "66aUo8zsujfVXRIT"
+    source = "https://services3.arcgis.com/66aUo8zsujfVXRIT/arcgis/rest/services/"
```

Comment:
@Tlyon3 this should point to a dashboard or web resource that somebody could visit to see/verify the data.

Could you please provide that link?